provisioning/atat_provisioning.yaml

```diff
@@ -77,139 +125,44 @@ paths:
           description: Invalid ID supplied
         '404':
           description: Portfolio not found
-  '/portfolios/{portfolioId}/cost':
+  '/portfolios/{portfolioId}/taskorders':
     post:
       tags:
         - cost
-      summary: Returns cost data by Portfolio
-      description: Queries CSP for actual or forecasted cost data by Portfolio
-      operationId: getCostsByPortfolio
+      summary: Adds a new funding source (TO)
+      description: Adds a new funding source to an existng Portfolio
+      operationId: addFundingSource
       parameters:
         - name: portfolioId
           in: path
-          description: ID of the portfolio
+          description: ID of the Portfolio
           required: true
           schema:
             type: string
-      requestBody:
-        required: true
-        content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/CostRequest'
-            examples:
-              CostRequestActual:
-                $ref: '#/components/examples/CostRequestActual'
       responses:
         '200':
-          description: Cost operation successful
+          description: Successful operation
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/CostResponse'
-              examples:
-                CostResponseActual:
-                  $ref: '#/components/examples/CostResponseActual'
+                $ref: '#/components/schemas/PortfolioOperatorRole'
```

Comment:
I might be missing something, but is the `200` response supposed to be a role for `POST /porfolios/{portfolioId}/taskorders`?

Reply:
Fixed in #11